packages/ckeditor5-table/src/tablecolumnresize/commands/tableresizecolumncommand.ts

```diff
@@ -0,0 +1,101 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+/**
+ * @module table/tablecolumnresize/commands/tableresizecolumncommand
+ */
+
+import { Command, type Editor } from 'ckeditor5/src/core.js';
+import type { DowncastInsertEvent } from 'ckeditor5/src/engine.js';
+import type { PossibleResizeColumnRange } from '../tablecolumnresizeutils.js';
+import { _getSmallestColumnResizer } from '../utils/getsmallestcolumnresizer.js';
+
+/**
+ * The resize table column command.
+ *
+ * The command is registered by {@link module:table/tablecolumnresize/tablecolumnresizeui~TableColumnResizeUI}
+ * as the `'resizeTableColumn'` editor command.
+ *
+ * To resize currently selected column, execute the command:
+ *
+ * ```ts
+ * editor.execute( 'resizeTableColumn', { newColumnWidth: 250 } );
+ * ```
+ */
+export default class TableResizeColumnCommand extends Command {
+	/**
+	 * The command value: Current size of column (in pixels) and possible resize range of such column..
+	 *
+	 * @readonly
+	 * @observable
+	 */
+	declare public possibleRange: PossibleResizeColumnRange | null;
+
+	constructor( editor: Editor ) {
+		super( editor );
+
+		// After inserting a table, the Resizer column elements are added to the document.
+		// However, the plugin remains disabled because none of the column resizers have been mounted in the DOM yet.
+		// To ensure that the resize option is enabled, refresh the plugin after rendering the table.
+		this.editor.conversion.for( 'editingDowncast' ).add( dispatcher => {
+			dispatcher.on<DowncastInsertEvent>( 'insert:table', () => {
+				editor.editing.view.once( 'render', this.refresh.bind( this ) );
+			} );
+		} );
+	}
+
+	/**
+	 * @inheritDoc
+	 */
+	public override refresh(): void {
+		const { plugins } = this.editor;
+
+		const resizeUtils = plugins.get( 'TableColumnResizeUtils' );
+		const tableUtils = plugins.get( 'TableUtils' );
+
+		const smallestResizer = _getSmallestColumnResizer( this.editor );
+
+		if ( smallestResizer ) {
+			const { last, first } = tableUtils.getColumnIndexes(
+				tableUtils.getSelectionAffectedTableCells( this.editor.model.document.selection )
+			);
+
+			if ( last === first ) {
+				this.isEnabled = true;
+				this.possibleRange = resizeUtils._getPossibleResizeColumnRange( smallestResizer );
+				return;
+			}
+		}
```

Comment:
This part requires a couple comments answering questions like: "why the heck we get the smallest resizer?" etc.

Reply:
Adjusted docs in `_getSmallestSelectedColumnResizer` util method. `resizer` term is used in many other places in this particular plugin, so imo it does not need further explanation.